components/drift/drift.app.mjs

```
@@ -1,11 +1,140 @@
+
+        try {
+          response = await this.getContactById({
+            $,
+            contactId,
+          });
+        } catch (error) {
+          if (error.status === 404) {
+            throw new Error(`No contact found with ID: ${contactId}`);
+          } else {
+            throw error;
+          };
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)